### ISSUE TITLE
Better error handling for channel opening under min

### ISSUE THREAD

**Issue by thedon-chris** [lightninglabs/lightning-app#1026]
### Description
There is no proper error handling for attempting to open channel under required amount. 

When attempting to open channel much under min,  lightning-app just displays "Creating channel failed!". 

This happened only when sending >20k sats, but when sending 20.5k sats another error showed.

Instead inform the user of minimum channel amount, and throw more informative errors.

### Steps to reproduce the behavior
Attempt to open channel under 20k sats.
Attempt to open channel with 20.1-20.5k
### Expected behavior
Throw informative error

### Actual behavior
Does not throw informative error

### Screenshot and/or logs if applicable
What it looks like now
![Screen Shot 2019-03-19 at 4 30 28 PM](https://user-images.githubusercontent.com/30728737/54639794-9942fd00-4a64-11e9-8ac1-2097fc3f3771.png)
What it should look like
![Screen Shot 2019-03-19 at 4 30 28 PM](https://user-images.githubusercontent.com/30728737/54639880-c98a9b80-4a64-11e9-9ff8-5e95087efa5e.png)

**Comment by tanx** [comment]
Good call. We should definitely improve error messages in notifications.
